src/InstallationValidator.Core/Extensions/ComparisonPresenterExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using InstallationValidator.Core.Assets;
+using InstallationValidator.Core.Presentation;
+using OSPSuite.Core.Extensions;
+
+namespace InstallationValidator.Core.Extensions
+{
+   public static class ComparisonPresenterExtensions
```

Comment:
I would not name that comparison view. This is more a IHtmlLoggerPresenter or sthg no?